src/Compat/FormDecorator/LabelDecorator.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace ipl\Web\Compat\FormDecorator;
+
+use ipl\Html\Attributes;
+use ipl\Html\Contract\FormElement;
+use ipl\Html\Contract\FormSubmitElement;
+use ipl\Html\Contract\HtmlElementInterface;
+use ipl\Html\FormDecorator\DecorationResults;
+use ipl\Html\HtmlElement;
+use ipl\Html\HtmlString;
+use ipl\Html\Text;
+use ipl\Html\FormDecorator\LabelDecorator as IplHtmlLabelDecorator;
+/**
+ * Decorates the label of the form element
+ */
+class LabelDecorator extends IplHtmlLabelDecorator
```

Comment:
At the moment, extending this class does not have any advantage. Please give it this one:

ipl\Html should provide a method to retrieve the form element label: `protected function getElementLabel(FormElement $formElement): ?ValidHtml`

Then this decorator can override it with:

```
        $label = parent::getElementLabel($formElement);
        if ($label === null) {
            return HtmlString::create('&nbsp;');
        }
```